packages/patternfly-3/patternfly-react/src/components/DateTimePicker/DateComponents/TodayButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TodayButton extends React.Component {
+  setToday = () => {
+    const { setSelected } = this.props;
+    if (setSelected) setSelected(new Date());
+  };
+  render() {
+    return (
+      <table className="table-condensed">
+        <tbody>
+          <tr>
+            <td>
+              <a onClick={this.setToday} title="Go to today">
```

Comment:
instead of an anchor `<a />`  you should use a button as this isn't a link, [read more](https://bitsofco.de/anchors-vs-buttons/)